src/dhcp/nm-dhcp-systemd.c

```diff
@@ -369,12 +370,40 @@ lease_to_ip4_config (NMDedupMultiIndex *multi_idx,
 
 	num = sd_dhcp_lease_get_routes (lease, &routes);
 	if (num > 0) {
-		nm_gstring_prepare (&str);
+		nm_auto_free_gstring GString *str_classless = NULL;
+		nm_auto_free_gstring GString *str_static = NULL;
+		guint32 default_route_metic = route_metric;
+
 		for (i = 0; i < num; i++) {
-			const char *gw_str;
+			switch (sd_dhcp_route_get_option(routes[i])) {
```

Comment:
Missing space before open parenthesis.

In the commit message title: intenal -> internal

Reply:
Fixed.